test/lru.ml

```diff
@@ -7,7 +7,21 @@ module Test (K : sig
   val pp : t Fmt.t
 end) =
 struct
-  module Lru = Cachecache.Lru.Make (K)
+  module Cache = Cachecache.Lru.Make (K)
+
+  module DB = struct
+    type value = K.t
+    type key = K.t
+    type t = (K.t, value) Hashtbl.t
+
+    let v () = Hashtbl.create 10
```

Comment:
`10` seems a bit arbitrary
```suggestion
    let v () = Hashtbl.create 0
```

Reply:
I agree, is 0 a accepted value ?

Reply:
Yes, `0` is accepted. In any case, `Hashtbl.create` will pick the next power of `2`, starting at `16` (see [here](https://github.com/ocaml/ocaml/blob/trunk/stdlib/hashtbl.ml#L73-L78)). So any value under `16` will result in `16` as the initial size.